src/Combobox.jsx

```diff
@@ -293,6 +304,13 @@ var ComboBox = React.createClass({
     this.refs.input.focus()
   },
 
+  handlePopupOpening() {
+    if (this.props.tetherPopup) {
+      compat.findDOMNode(this.refs.list).focus();
```

Comment:
Added this line because, if the popup list is scrollable, scrolling to the bottom and clicking on an item causes it to scroll to the top instead of selecting the item.